### PR TITLE
fix CloneElement Porps

### PR DESCRIPTION
**Pull request by myoneflag**

### CODE REVIEW

File: src/google_map_markers.js [hunk 1]
```diff
@@ -263,10 +263,10 @@ export default class GoogleMapMarkers extends Component {
           child.props.lng === undefined
         ) {
           return React.cloneElement(child, {
-            $geoService: this.props.geoService,
-            $onMouseAllow: this._onMouseAllow,
-            $prerender: this.props.prerender,
```
Comment (itsmichaeldiego) [inline src/google_map_markers.js:268]: Why are we removing the `$` prefix?

**Review by itsmichaeldiego — COMMENTED** [review]
Wondering what's the issue you're facing and how does this fix it, could you provide more info?

File: src/google_map_markers.js (hunk 1 above)
Comment (myoneflag) [reply src/google_map_markers.js:268]: Sorry, my misunderstand. I am very interested in this cooperation. Is there any issue let me fix?

Comment (myoneflag) [reply src/google_map_markers.js:268]: Sorry again

**Pull request closed without merging**